nav2_velocity_smoother/src/velocity_smoother.cpp

```diff
@@ -209,6 +209,11 @@ double VelocitySmoother::applyConstraints(
 
 void VelocitySmoother::smootherTimer()
 {
+  // Wait until the first command is received
+  if (command_==nullptr) {
```

Comment:
```suggestion
  if (!command_) {
```

Reply:
Fixes both a linting issue and a style issue

Reply:
Thank you!